internal/cue/validate.go

```diff
@@ -130,6 +130,11 @@ func ValidateFiles(dst io.Writer, files []string, format string) error {
 
 			for _, m := range ce {
 				ips := m.InputPositions()
+				fmt.Printf("Pos: [line: %d, col: %d]\n", m.Position().Line(), m.Position().Column())
```

Comment:
I take it `m.Position()` doesnt return the right thing either?

Reply:
sadly it doesnt seem to, line and col are both 0